samples/js/js-eventsourced-shopping-cart/proto/shoppingcart_api.proto

```diff
@@ -13,15 +13,28 @@
 // limitations under the License.
 
 // This is the public API offered by the shopping cart entity.
-
+/* tag::declarations[] */
```

Comment:
I thought asciidoc required the tags to be at the end of the line. May be good to check that this works, compared with:

```proto
// tag::declarations[]
```

Reply:
I've copied these tags verbatim from the current doc sources (https://github.com/lightbend/akkaserverless-docs/blob/master/docs/modules/src/examples/js-shopping-cart/shoppingcart_api.proto) to make sure that I used the same tags I had before (which made it easier for me to update the quickstart 😅). After rendering, it comes out as https://developer.lightbend.com/docs/akka-serverless/quickstart/cr-value-entity-javascript.html. With this PR we can remove the final code snippets from the docs and make sure all samples in the docs come from the SDK repos.

Reply:
Ok, cool

Reply:
Just to keep the style the same, I updates the tags to `// tag`